### DISCUSSION TITLE
BirdCam Settings

### DISCUSSION THREAD

**Discussion by kabakakao** [alienatedsec/yi-hack-v5#357] [Q&A]
I am figuring out, what the best settings for a Bird cam will be.
I am running the cam with a 20.000 mAh Powerbank. But with this configuration the Cam just works for about 3 days.

I already disabled Rtsp and onvif.

I don‘t need live access to the cam, so wifi could be disabled in normal cases. It would be nice, if there is a regular „Health“ mqtt message and maybe a snapshot and then it deactivates WIFI to Save power
On motion it should record to sd card.

To reactivate Standard wifi Connection I would prefer to touch a file to the SD card

what do you think about that?

**Comment by alienatedsec** [comment]
@kabakakao sounds like an interesting project; however, why not try alternative power sources e.g. A mini Solar system with battery storage? You could even use the same powerbank. 

Disabling wifi does not give much power saving.

**Reply by kabakakao** [reply]
I have already a Solar Panel in Place. But the consumption of the cam is too high (at least in Winter and Spring)
Are there any ideas to lower the consumption of the cam?

**Comment by premiero** [comment]
Interestingly, I am using my Yi cam (27US) as a bird cam too (Sugar glider nesting box). 

![obraz](https://github.com/alienatedsec/yi-hack-v5/assets/27424909/41350328-0e20-4b59-b11f-be3d62531f68)

But I am using it with regular power supply, just added 5m of USB extender to reach my power outlet socket, wchich I installed outside the building.
@alienatedsec From my knowledge disabling Wi-Fi does have major impact on battery life, at least in the case of smartphones.

As for settings - is there any way to change the switching behaviour of day/night vision? I need bigger hysteresis for that. Sometimes camara just switches between day and night for an hour cllicking all the way and stressing birds. Maybe good idea would be adding manual day/night mode to firmware?
